README.md

```diff
@@ -87,11 +87,16 @@ company
 function `capitalize`
 
 ```ts
-company
+const capitalizeName = company
   .compose(address)
   .compose(street)
   .compose(name)
-  .modify(capitalize)(employee)
+  .modify(capitalize)
+  
+assert.deepEqual(
```

Comment:
Nitpick: I'd use `deepStrictEqual` here (AFAIK `deepEqual` is deprecated) and the second argument is supposed to be the expected value, so

```ts
assert.deepStrictEqual(
  capitalizeName(employee),
  employeeCapitalized,
)
```